index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```

Comment:
can you change the title to something like: `Pepsi webpage`?

```suggestion
    <title>Pepsi webpage</title>
```